### PR TITLE
pkg/cluster: remove disaster recovery/restore and backupmanager

### PR DESCRIPTION
**Pull request by fanminshi**
ref: https://github.com/coreos/etcd-operator/issues/1626

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

File: pkg/cluster/reconcile.go [hunk 1]
```diff
@@ -90,8 +89,8 @@ func (c *Cluster) reconcileMembers(running etcdutil.MemberSet) error {
 	}
 
 	if L.Size() < c.members.Size()/2+1 {
-		c.logger.Infof("Disaster recovery")
-		return c.disasterRecovery(L)
+		c.logger.Infof("lost quorum")
+		return fmt.Errorf("lost quorum")
```
Comment (hongchaodeng) [inline pkg/cluster/reconcile.go]: `errors.New("lost quorum")`

Make it a global var.

**Comment by hongchaodeng** [comment]
Test failing.

**Comment by fanminshi** [comment]
fmt check was complaining that some constants in backupmanager is not used. so i remove backupmanager in this pr too.

**Comment by hongchaodeng** [comment]
LGTM

**Pull request merged**